subsys/bluetooth/host/conn_internal.h

```diff
@@ -26,28 +26,29 @@ typedef enum __packed {
 /* bt_conn flags: the flags defined here represent connection parameters */
 enum {
 	BT_CONN_AUTO_CONNECT,
-	BT_CONN_BR_LEGACY_SECURE,	/* 16 digits legacy PIN tracker */
-	BT_CONN_USER,			/* user I/O when pairing */
-	BT_CONN_BR_PAIRING,		/* BR connection in pairing context */
-	BT_CONN_BR_NOBOND,		/* SSP no bond pairing tracker */
-	BT_CONN_BR_PAIRING_INITIATOR,	/* local host starts authentication */
-	BT_CONN_CLEANUP,                /* Disconnected, pending cleanup */
-	BT_CONN_PERIPHERAL_PARAM_UPDATE,/* If periph param update timer fired */
-	BT_CONN_PERIPHERAL_PARAM_SET,	/* If periph param were set from app */
-	BT_CONN_PERIPHERAL_PARAM_L2CAP,	/* If should force L2CAP for CPUP */
-	BT_CONN_FORCE_PAIR,             /* Pairing even with existing keys. */
+	BT_CONN_BR_LEGACY_SECURE,	      /* 16 digits legacy PIN tracker */
+	BT_CONN_USER,			      /* user I/O when pairing */
+	BT_CONN_BR_PAIRING,		      /* BR connection in pairing context */
+	BT_CONN_BR_NOBOND,		      /* SSP no bond pairing tracker */
+	BT_CONN_BR_PAIRING_INITIATOR,	      /* local host starts authentication */
+	BT_CONN_CLEANUP,                      /* Disconnected, pending cleanup */
+	BT_CONN_PERIPHERAL_PARAM_UPDATE,      /* If periph param update timer fired */
+	BT_CONN_PERIPHERAL_PARAM_AUTO_UPDATE, /* If periph param auto update on timer fired */
+	BT_CONN_PERIPHERAL_PARAM_SET,	      /* If periph param were set from app */
+	BT_CONN_PERIPHERAL_PARAM_L2CAP,	      /* If should force L2CAP for CPUP */
+	BT_CONN_FORCE_PAIR,                   /* Pairing even with existing keys. */
 #if defined(CONFIG_BT_GATT_CLIENT)
-	BT_CONN_ATT_MTU_EXCHANGED,	/* If ATT MTU has been exchanged. */
+	BT_CONN_ATT_MTU_EXCHANGED,	      /* If ATT MTU has been exchanged. */
 #endif /* CONFIG_BT_GATT_CLIENT */
 
-	BT_CONN_AUTO_FEATURE_EXCH,	/* Auto-initiated LE Feat done */
-	BT_CONN_AUTO_VERSION_INFO,      /* Auto-initiated LE version done */
+	BT_CONN_AUTO_FEATURE_EXCH,	      /* Auto-initiated LE Feat done */
+	BT_CONN_AUTO_VERSION_INFO,            /* Auto-initiated LE version done */
 
-	BT_CONN_CTE_RX_ENABLED,          /* CTE receive and sampling is enabled */
-	BT_CONN_CTE_RX_PARAMS_SET,       /* CTE parameters are set */
-	BT_CONN_CTE_TX_PARAMS_SET,       /* CTE transmission parameters are set */
-	BT_CONN_CTE_REQ_ENABLED,         /* CTE request procedure is enabled */
-	BT_CONN_CTE_RSP_ENABLED,         /* CTE response procedure is enabled */
+	BT_CONN_CTE_RX_ENABLED,               /* CTE receive and sampling is enabled */
+	BT_CONN_CTE_RX_PARAMS_SET,            /* CTE parameters are set */
+	BT_CONN_CTE_TX_PARAMS_SET,            /* CTE transmission parameters are set */
+	BT_CONN_CTE_REQ_ENABLED,              /* CTE request procedure is enabled */
+	BT_CONN_CTE_RSP_ENABLED,              /* CTE response procedure is enabled */
```

Comment:
Since you have the "fix tab usage in struct" in a separate cleanup patch, I think this part should also be moved away and not be part of a functional change patch. Btw, even with your changes, there seems to be a mix of tabs and spaces for some of the enum values between the `,` and the `/*`

Reply:
Agreed. 

IIRC, tabs should only be used for the initial indentation, and any indentation after a non-tab character should use space. 

It is fine to update the indentation, but should be done correctly, and as Johan mentioned, may be a good idea to do in a non-functional change patch.

Reply:
> Since you have the "fix tab usage in struct" in a separate cleanup patch, I think this part should also be moved away and not be part of a functional change patch. Btw, even with your changes, there seems to be a mix of tabs and spaces for some of the enum values between the `,` and the `/*`

Fixed now (enum had appeared aligned in my editor and hence did not notice the tabs when adding the new flag). There are other structs in the file with tabs, they should be addressed in the future as separate PRs.